ui/src/overridden/backoffice/Importer/ImportedDocuments.js

```diff
@@ -0,0 +1,200 @@
+
+const importerURL = '/importer/check';
+
+export class ImportedDocuments extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeIndex: -1,
+      importCompleted: false,
+      data: null,
+      intervalId: null,
+      isLoading: true,
+    };
+  }
+
+  componentDidMount() {
+    //TODO: Define how much we wait until the next call (currently 30 sec)
+    const idVar = setInterval(() => this.checkForData(idVar), 30000);
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ intervalId: idVar });
+    this.checkForData(idVar);
+    window.addEventListener('beforeunload', this.onCloseWarning);
+  }
+
+  componentWillUnmount = () => {
+    const { intervalId } = this.state;
+    window.removeEventListener('beforeunload', this.onCloseWarning);
+    clearInterval(intervalId);
+  };
+
+  onCloseWarning = e => {
+    const { isLoading } = this.state;
+    if (isLoading) {
+      var confirmationMessage =
+        'Importing in progress. Are you sure you want to leave?';
+
+      (e || window.event).returnValue = confirmationMessage; //Gecko + IE
+      return confirmationMessage; //Webkit, Safari, Chrome
+    }
+  };
+
+  checkForData = async idVar => {
+    const { importCompleted } = this.state;
+    const { taskId } = this.props;
+
+    if (!importCompleted) {
+      // eslint-disable-next-line no-useless-concat
+      const response = await http.get(`${importerURL}` + '/' + `${taskId}`);
+      if (response.data.state !== 'RUNNING') {
+        this.setState({
+          importCompleted: true,
+          isLoading: false,
+          data: response.data,
+        });
+      } else {
+        this.setState({
+          data: response.data,
+          isLoading: true,
+        });
+      }
+    } else {
+      clearInterval(idVar);
+    }
+  };
+
+  handleClick = (e, titleProps) => {
+    const { index } = titleProps;
+    const { activeIndex } = this.state;
+    const newIndex = activeIndex === index ? -1 : index;
+
+    this.setState({ activeIndex: newIndex });
+  };
+
+  renderErrorMessage = data => {
+    return (
+      <Message negative>
+        <Message.Header>Failed to import</Message.Header>
+        <p>
+          The import of the documents failed, please try again. <br />
+          If this error persists contact our technical support.
+        </p>
+      </Message>
+    );
+  };
+
+  render() {
+    const { data, activeIndex, isLoading } = this.state;
+    const { cleanData } = this.props;
+    return (
+      <>
+        {isLoading || !data ? (
+          <>
+            <Icon loading name="circle notch" />
+            Importing documents... This may take a while.
+          </>
+        ) : data.state === 'SUCCEEDED' ? (
+          <>
+            <Icon name="check" />
+            Documents imported succesfully.
+          </>
+        ) : (
+          <>
+            <Icon name="close" />
+            Documents import failed.
```

Comment:
It can be also Series and EItems? If yes, I would use Literature everywhere.

Reply:
If I am not wrong we import documents, but we also create/update/delete eitems related to them, so yes we can use Literature